src/scripts/build-api-pages.js

```diff
@@ -786,11 +786,20 @@ const processSpecs = (specs) => {
           const jsonString = safeJsonStringify(deref, null, 2);
           const pathToJson = `./data/api/${version}/full_spec_deref.json`;
           fs.writeFileSync(pathToJson, jsonString, 'utf8');
+
           // make a copy in static for postman
-          fs.copyFile(pathToJson, `./static/resources/json/full_spec_${version}.json`, (err) => {
-            if (err) throw err;
-            console.log(`full_spec_${version}.json copied to /static/resources/json/`);
-          });
+          // the postman copy needs to not include the empty "tags" that we
+          // included to ensure redirection in the docs page from v2 <-> v1
+          var i;
+          var derefStripEmptyTags = lodash.cloneDeep(deref);
+          for (i = derefStripEmptyTags.tags.length - 1; i >= 0; i -= 1) {
+              if (derefStripEmptyTags.tags[i].description.toLowerCase().includes("see api version")) {
```

Comment:
This seemed to be the most straightforward way of detecting empty tags without iterating over the entire schema to see if the tag was used.